css/view.css

```diff
@@ -3,6 +3,15 @@
   display: none !important;
 }
 
-span.tvm__text:not(:has(span)) {
+/* hides "see how you compare to x applicants" option next to reactivate premium button */
```

Comment:
Thought I would add comments in the code to make it more clear what code affects what. I hope to go back and do the same thing to the search.css file too.